app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+from app.errors import (VaccineError, NotWearingMaskError)
+from app.cafe import Cafe
+
+
+def go_to_cafe(friends: list, cafe: "Cafe") -> str:
+    not_wearing_mask_count = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError:
+            return "All friends should be vaccinated"
```

Comment:
The function currently returns immediately upon encountering a `VaccineError`, which prevents checking all friends. Modify the logic to ensure all friends are checked before returning, as per the task requirements.